README.md

```diff
@@ -5,26 +5,56 @@ Python client library for [VinylDNS](https://www.vinyldns.io/)
 This project is a work in progress! If you would like to help us get this where it needs to be,
 please reach out to us in [gitter](https://gitter.im/vinyldns/Lobby).
 
-To run, `pip install vinyldns-python` and then:
+To run, `pip install vinyldns` and then:
 
 ```python
->>> from vinyldns_python import *
->>> local_client = client.VinylDNSClient("ApiEndpoint", "UserAccessKey", "UserSecretKey")
+>>> from vinyldns.client import VinylDNSClient
+>>> local_client = VinylDNSClient("ApiEndpoint", "UserAccessKey", "UserSecretKey")
```

Comment:
Note, the namespace and packaging are significant.

We _should not_ use `vinyldns_python` as the namespace / packaging, as this is a python library, the `python` bit is super redundant.